### PR TITLE
"QUAD-86:Designed new layout for the Public page settings and included a

### PR DESCRIPTION
**Pull request by prasanthpriyanesan**
new navigation page"

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/validator/AddProjectValidator.java [hunk 1]
```diff
@@ -31,7 +31,6 @@ public boolean supports(Class<?> arg0) {
      * 
      * @param obj
      * @param err
-     * @author Karthikeyan Mohan
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/validator/AddProjectValidator.java:34]: why did you remove that?

File: Quadriga/src/main/java/edu/asu/spring/quadriga/validator/PublicPageValidator.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+import edu.asu.spring.quadriga.domain.impl.workbench.Project;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+
+@Service
+public class PublicPageValidator implements Validator {
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/validator/PublicPageValidator.java:15]: Did you change or add that class?

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import edu.asu.spring.quadriga.domain.IUser;
+import edu.asu.spring.quadriga.domain.factory.workbench.IProjectFactory;
+import edu.asu.spring.quadriga.domain.impl.workbench.Project;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.IUserManager;
+import edu.asu.spring.quadriga.service.workbench.IModifyProjectManager;
+import edu.asu.spring.quadriga.validator.PublicPageValidator;
+
+@Controller
+public class PublicPageController {
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java:30]: Needs javadoc

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java:30]: I'm confused, why does the PublicPageController has method to create a new project?

File: Quadriga/src/main/java/edu/asu/spring/quadriga/validator/AddProjectValidator.java (hunk 1 above)
Comment (prasanthpriyanesan) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/validator/AddProjectValidator.java:34]: I had to remove it on PublicpageValidator.java file. But, accidentally I did on this java file as well, I will add it back.

File: Quadriga/src/main/java/edu/asu/spring/quadriga/validator/PublicPageValidator.java (hunk 2 above)
Comment (prasanthpriyanesan) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/validator/PublicPageValidator.java:15]: This is a new class we added.

**Comment by jdamerow** [comment]
Let's talk about this on Friday. There are a couple of things that confuse me.

**Comment by jdamerow** [comment]
also, there are conflicts

**Comment by jdamerow** [comment]
@prasanthpriyanesan, please resolve conflicts
